.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,90 @@
+<!-- Type of change
+Please label this PR with one of the following labels, depending on the scope of your change:
+- bug
+- enhancement
+- breaking change
+-->
```

Comment:
Include `doc` label?  We do a lot of docs-only PRs that might document but not fix a `bug`, for example.  Docs changes are more likely to be corrections or clarifications rather than `enhancements`. 

If we do add `doc`, would it make sense to allow contributors to label with "one or more?"